src/pages/Product/ProductCard/ProductCard.scss

```
@@ -0,0 +1,107 @@
+@import '../../../styles/common.scss';
+@import '../../../styles/variables.scss';
+
+.ProductCard {
+  // min-width: 415px;
+  margin: 0 10px 20px 0;
+  padding: 30px 30px;
+  border-radius: 10px;
+  box-shadow: 0 5px 4px 0 rgba(0, 0, 0, 0.05);
+  background-color: lemonchiffon;
+  &:nth-child(2n) {
+    margin-right: 0;
+  }
+
+  .productCardHeader {
+    @include display-flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    height: 110px;
+
+    h2 {
+      margin-top: 8px;
+
+      strong {
+        font-size: 20px;
+        font-weight: bold;
+        line-height: 30px;
+      }
+    }
+
+    .icon {
+      display: flex;
+      margin-top: 5px;
+    }
+  }
+
+  .pillImgBox {
+    display: flex;
+    align-self: center;
+
+    .pillImage {
+      width: 100px;
+      height: 50px;
+    }
+  }
+
+  .productCardBody {
+    @include display-flex;
+    justify-content: space-between;
+    height: 72px;
+
+    .quantityAndpriceBox {
+      height: 100%;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+      align-items: flex-end;
+      font-weight: bold;
+    }
+
+    .quantity {
+      font-size: 14px;
+    }
+
+    .price {
+      font-size: 20px;
+    }
+  }
+
+  .productCardFooter {
+    display: flex;
+    flex-direction: column;
+    margin-top: 14px;
+
+    .add {
+      margin-bottom: 20px;
+      font-size: 14px;
+      font-weight: bold;
+      color: #707070;
+      text-decoration: underline;
+    }
+
+    .cartBtn {
+      width: 300px;
+      height: 60px;
+      margin: 0 auto;
+      border: none;
+      border-radius: 30px;
+      background-color: #ffffff;
+      font-size: 16px;
+      font-weight: bold;
```

sass에서는 이런식으로도 적을수 있습니다 
```suggestion
      font:{
        size:16px;
        font-weight:bold;
      }
```

---

우와 이렇게까지 네스팅이되다니.. sass 대단합니다...! 

```
font: {
        size: 16px;
        weight: bold;
      }
```

적용해보았습니다! :) 👍🏻